Source/ResponseSerialization.swift

```diff
@@ -109,7 +109,7 @@ extension DataRequest {
     ///   - completionHandler: The code to be executed once the request has finished.
     /// - Returns:             The request.
     @discardableResult
-    public func response(queue: DispatchQueue? = nil, completionHandler: @escaping (DataResponse<Data?>) -> Void) -> Self {
+    public func response(queue: DispatchQueue = .main, completionHandler: @escaping (DataResponse<Data?>) -> Void) -> Self {
```

Comment:
If we're going to make this change, it would be good to audit all API that takes optional values with a coalesced value in the body, and change them all to work like this.